### DISCUSSION TITLE
Shelly plus 2PM non visibile su Haa Home manager

### DISCUSSION THREAD

**Discussion by FrankDem74** [RavenSystem/esp-homekit-devices#2210] [HAA Home App]
Sto programmando il mio Shelly 2PM, ho seguito tutte le indicazioni e sono arrivato ad inserire lo script.

Inserisco Sid wifi e password compreso lo script ma il dispositivo sparisce, non riesco più ad accedere perchè non lo vedo. 
Ho scaricato l'app Haa MAnager ma non vedo lo shelly per capire se si sta aggiornando.

Per poter riccedere devo resettarlo .

Cosa posso fare???

**Comment by FrankDem74** [comment]
### _helly Plus 2PM | ESP32 / ESP32_1 before 2022Q2 | 

Switch with external switches, power monitoring, and internal temperature monitoring from HAA Manager | Grouped. Auto power off when 75°C are reached |_ 

Ho inserito questa stringa

{"c":{"io":[[[12,13,0],2],[[4],6,1],[[5,18],6,0,1],[[35],10,0,3],[[33]]],"l":0,"b":[[4,5]],"ic":[[25,26,100]]},"a":[{"0":{"r":[[13]]},"1":{"r":[[13,1]]},"b":[[5],[5,0]],"es":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[18],[18,0]]},{"t":75,"cd":1.5,"n":2,"dt":[0,56],"vf":0.0000382602,"vo":-0.068,"cf":0.00000949523,"co":-0.017,"pf":0.006097560976},{"t":75,"n":3,"dt":[0,56],"vf":0.0000382602,"vo":-0.068,"cf":0.00000949523,"co":-0.017,"pf":0.006097560976},{"t":22,"h":2,"n":5,"g":35,"j":10,"y0":[{"v":75,"r":1,"0":{"m":[[1],[2]]}}]}]}]}
-- | -- | -- | -- | --

**Comment by spiccinocchi** [comment]
C’è qualche errore nello script e lo shelly si riavvia in setup mode. 
Nell’app HAA Manager c’è una funzione di setup radar che te lo mostra e ti consente di accedere e inserire lo script corretto che dovrebbe essere questo:

{"c":{"io":[[[12,13,0],2],[[4],6,1],[[5,18],6,0,1],[[35],10,0,3],[[33]]],"l":0,"b":[[4,5]],"ic":[[25,26,100]]},"a":[{"0":{"r":[[13]]},"1":{"r":[[13,1]]},"b":[[5],[5,0]],"es":[{"0":{"r":[[12]]},"1":{"r":[[12,1]]},"b":[[18],[18,0]]},{"t":75,"cd":1.5,"n":2,"dt":[0,56],"vf":0.0000382602,"vo":-0.068,"cf":0.00000949523,"co":-0.017,"pf":0.006097560976},{"t":75,"n":3,"dt":[0,56],"vf":0.0000382602,"vo":-0.068,"cf":0.00000949523,"co":-0.017,"pf":0.006097560976},{"t":22,"h":2,"n":5,"g":35,"j":10,"y0":[{"v":75,"r":1,"0":{"m":[[1],[2]]}}]}]}]}

Con quale firmware haa lo hai programmato?

**Comment by FrankDem74** [comment]
Ciao lo shelly ha smesso di funzionare, nemmeno tutte le operazioni possibili di reset hanno permesso il riavvio.

Ho utilizzato mgos32-to-tasmota32-Plus2PM e sucessivamente haaboot..esp32_1


Non so se fidarmi a farlo su un dispositivo nuovo anche perché ho scoperto di poterlo comandare con Siri attraverso l’app Home

**Reply by spiccinocchi** [reply]
Se disponi del programmatore UART-USB se lo programmi con il firmware fullhaaboot_esp32_1.bin non avrai problemi ad utilizzarlo in homekit

**Comment by FrankDem74** [comment]
Grazie mille del consiglio ma sono un novellino, ho provato via OTA non ho mai usato il programmatoreFrancesco DemarcusIl giorno 13 feb 2024, alle ore 16:16, Simone Piccinocchi ***@***.***> ha scritto:﻿
Se disponi del programmatore UART-USB se lo programmi con il firmware fullhaaboot_esp32_1.bin non avrai problemi ad utilizzarlo in homekit
